instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/__init__.py

```diff
@@ -191,11 +193,14 @@ def _trace_postrun(self, *args, **kwargs):
         logger.debug("postrun signal task_id=%s", task_id)
 
         # retrieve and finish the Span
-        span, activation = utils.retrieve_span(task, task_id)
-        if span is None:
+        ctx = utils.retrieve_context(task, task_id)
+
+        if ctx is None:
```

Comment:
before the span was checked, now only if the entry is found. Is it on purpose?

Reply:
before the function would return `(None, None)` when the entry was not found, but there was no case where the span had a value but the activation value did not. To make it a bit clearer I made the function return either the tuple with values or a single None.

Functionally they behave the same.